spec/controllers/api/v2/certificates_controller_index_spec.rb

```diff
@@ -34,12 +34,16 @@
 
     it { is_expected.to match_json_expression(pattern) }
 
-    it 'the TimeMachine receives the correct Date' do
-      allow(TimeMachine).to receive(:at).and_call_original
+    context 'when the validity_end_date is set to a past date' do
+      let(:certificate) { create(:certificate, validity_end_date: 1.day.ago) }
 
-      do_response
+      let(:empty_response_pattern) do
```

Comment:
This could be a 1 liner instead of 5?

Reply:
yeah, actually we can even fully get rid of the let.